frontend/packages/dev-console/console-extensions.json

```diff
@@ -725,6 +740,24 @@
       "required": ["OPENSHIFT_BUILDCONFIG"]
     }
   },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "software-catalog",
+      "perspective": "admin",
+      "section": "home",
+      "name": "%devconsole~Software Catalog%",
```

Comment:
I worry that this will be confused with OperatorHub, although I know this is in @kevinhatchoua's mock. This is the `+Add` page on the dev side, correct? I also worry that we'll have different labels in the admin and dev perspectives when they're both enabled.

Reply:
No, it is not a `+Add` page. It is a Catalog page showing all the catalogs of 'Builder Images`, `DevFiles`, `Templates`, `Event Sources`, etc. We should have the same labels in both perspectives, as there is no difference in content. Software Catalog in Admin and Developer Catalog in the Dev perspective.